Add new sensor test

jianliu617 opened the pull request:
1) Add Intel Copyright and MIT license header;
2) Add OffsetTestExpectedValue for ORIENTATIONGEOMAGNETIC;
3) Add frequency, offset, jitter, drift and packet loss tests for geomagnetic orientation sensor;
4) Change ALS precondition test instruction for offset and jitter;
5) Fix a bug in line 857: Sensor type error. It should be orientation, not gyrometer;

jianliu617 committed:
Update Constants.cs

1) Add Intel Copyright and MIT license header;
2) Add OffsetTestExpectedValue for ORIENTATIONGEOMAGNETIC

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

1) Add frequency, offset, jitter, drift and packet loss tests for geomagnetic orientation sensor;
2) Change ALS precondition test instruction for offset and jitter;
3) Fix a bug in line 857: Sensor type error. It should be orientation, not gyrometer;

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

Fix build error in DisplayPrecondition();

Yongjie2018 commented on sensors/Tools/SensorExplorer/code/SensorExplorer/Scenario0_Tests.xaml.cs:
```diff
@@ -292,14 +297,28 @@ private void DisplayPrecondition()
+                        instruction.Text = "Please put the device on static level with its ambient light sensor covered.\n";
+                    }
+                    else
+					{
```
indent should be in space instead of tab.

Yongjie2018 commented on sensors/Tools/SensorExplorer/code/SensorExplorer/Scenario0_Tests.xaml.cs:
```diff
@@ -292,14 +297,28 @@ private void DisplayPrecondition()
-                                       "Keep it in stationary state.";
+                    if(type == Sensor.LIGHTSENSOR)
+                    {
+                        instruction.Text = "Please put the device on static level with its ambient light sensor covered.\n";
```
Suggest on wording: "Put device on a static level surface, with its ambient light sensor covered.\n"

Yongjie2018 commented on sensors/Tools/SensorExplorer/code/SensorExplorer/Scenario0_Tests.xaml.cs:
```diff
@@ -292,14 +297,28 @@ private void DisplayPrecondition()
-                                       "Keep it in stationary state.";
+                    if(type == Sensor.LIGHTSENSOR)
+                    {
+                        instruction.Text = "Please put the device on static level with its ambient light sensor covered.\n";
```
Suggest on wording: "Put device on a static level surface, with its ambient light sensor covered.\n"

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

Update based on @Yongjie2018 's commment:
indent should be in space instead of tab.

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

Update based on @Yongjie2018 @RuilinZhuIntel 's comments to update wordings.

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

Update a tab to space.

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

Remove tab indent.

jianliu617 committed:
Update Scenario0_Tests.xaml.cs

Remove "Please" in Precondition instruction.
